luatest/comparator.lua

```diff
@@ -70,6 +70,31 @@ function comparator.is_subset(actual, expected)
     return #expected_array - found_count
 end
 
+-- Checks that any element from the actual is an element from the expected.
+function comparator.any(actual, expected)
```

Comment:
The function should probably be called `intersects`.

Reply:
Reworked the function a bit. It is now called `are_disjoint()` and returns `false` if at least one of the arguments is not a table or if there is a common element for both arguments. Otherwise it returns `true`.